gradle/findbugs-exclude.xml

```diff
@@ -305,4 +305,16 @@ For a detailed description of findbugs bug categories, see http://findbugs.sourc
         <Source name="RoundTripWorker.java"/>
         <Bug pattern="NN_NAKED_NOTIFY"/>
     </Match>
+
+    <Match>
+        <Package name="org.apache.kafka.streams.scala"/>
+        <Source name="FunctionConversions.scala"/>
+        <Bug pattern="EQ_UNUSUAL"/>
+    </Match>
+
+    <Match>
+        <Package name="org.apache.kafka.streams.scala"/>
+        <Source name="ScalaSerde.scala"/>
+        <Bug pattern="NM_SAME_SIMPLE_NAME_AS_INTERFACE"/>
```

Comment:
This seems to be a false positive.  FindBugs is reporting that `Serializer` and `Deserializer` should be defined as a different type name then what it's inheriting.  IIRC the consensus earlier is that we want type names the same as the base types they're wrapping (which includes traits and interfaces IMO).  I've updated the FindBugs rule exclusion to be specific to the types generating the violation, rather than the entire `ScalaSerde` file.